src/app/formFields/getAddressValidationSchema.spec.tsx

```diff
@@ -1,13 +1,12 @@
 import { createLanguageService, LanguageService } from '@bigcommerce/checkout-sdk';
 import { ObjectSchema, ValidationError } from 'yup';
 
+import { getFormFields } from '../address/formField.mock';
 import { getShippingAddress } from '../shipping/shipping-addresses.mock';
 
-import { getFormFields } from './formField.mock';
-import getAddressValidationSchema from './getAddressValidationSchema';
-import { AddressFormValues } from './mapAddressToFormValues';
+import { default as getFormFieldsValidationSchema, FormFieldValues } from './getFormFieldsValidationSchema';
 
-describe('getAddressValidationSchema', () => {
+describe('getFormFielsValidationSchema', () => {
```

Comment:
Same with this spec file. It is still called `getAddressValidationSchema.spec.tsx`.